### ISSUE TITLE
Show values for alerting queries

### ISSUE THREAD

**Issue by Thor77** [Vonage/Grafana_Status_panel#87]
First of all, thanks a lot for this great plugin, it's great to check the status of an entire infrastructure at a quick glance!

Currently values of queries are only shown, if the toggle to show a query all the time (even if they're not below/above configured thresholds) is checked.
Please consider adding an option (or set it as the default) to show the values of alerting queries besides their alias instead of just the alias.
Additionally it would be a nice feature to highlight the queries, setting a tiles color to red/orange, somehow (especially as i'm currently forced to check "Show always" for all queries to see their values).

**Comment by YehonatanVonage** [comment]
Hi Thor,
Thanks for your feedback. I didn't understand your 2nd suggestion, can you rephrase it?

**Comment by Thor77** [comment]
@YehonatanVonage 
If you set *Threshold Coloring > Coloring* to *Panel* **and** check *Threshold > Show Always* on all queries, you can't see which query is the one hitting a threshold.
I suggested to highlight the query hitting a threshold in some way to distinguish it from the ones not hitting a threshold.

Once my first problem/suggestion is solved, though, I don't really see a reason to check *Show Always* for all metrics so that wouldn't be a concern (at least for me) anymore.

**Comment by YehonatanVonage** [comment]
Both issues fixed in version 1.0.7: 
https://github.com/Vonage/Grafana_Status_panel/releases/tag/1.0.7

**Comment by Thor77** [comment]
Thanks :)
